packages/environment-check/src/checks/get-installed.ts

```diff
@@ -159,6 +159,11 @@ export async function getFioriGenVersion(): Promise<string> {
  * @returns modules and versions
  */
 export async function getProcessVersions(): Promise<NodeJS.ProcessVersions> {
-    const output = await spawnCommand('node', ['-p', 'JSON.stringify(process.versions)']);
-    return JSON.parse(output);
+    try {
+        const output = await spawnCommand('node', ['-p', 'JSON.stringify(process.versions)']);
+        return JSON.parse(output);
+    } catch (e) {
+        console.error(t('error.retrievingProcessVersions', { error: e.message }));
```

Comment:
@cianmSAP, can you please pass the logger into this function and log the error there, instead of the `console` so it is visible in the results? I think for an environment check it would be helpful to know why execution of command 
```
node -p 'JSON.stringify(process.versions)'
``` 
fails.

Reply:
Updated